src/server/server_family.cc

```diff
@@ -1651,9 +1650,11 @@ GenericError ServerFamily::DoSaveCheckAndStart(bool new_version, string_view bas
 }
 
 GenericError ServerFamily::WaitUntilSaveFinished(Transaction* trans, bool ignore_state) {
+  VLOG(1) << "WaitUntilSaveFinished";
   save_controller_->WaitAllSnapshots();
   detail::SaveInfo save_info;
 
+  VLOG(1) << "Before WaitUntilSaveFinished::Finalize";
```

Comment:
Two vlogs be a little noisy 🤷🏻‍♂️

Reply:
these are printed are only during saves and it's a vlog :)